keel-sql/src/main/kotlin/com/netflix/spinnaker/keel/sql/SqlVerificationRepository.kt

```diff
@@ -117,6 +118,25 @@ class SqlVerificationRepository(
     }
   }
 
+  override fun wasSuccessfullyVerifiedIn(
+    deliveryConfig: DeliveryConfig,
+    artifactReference: String,
+    version: String,
+    environmentName: String,
```

Comment:
Can we re-use the `VerificationContext` object like on the other methods' signatures?

Reply:
Done.